cryptodoc/src/08_signatures.rst

```diff
@@ -334,15 +334,17 @@ The signature generation algorithm works as follows:
    3. Compute point :math:`W=(x_1,y_1)=k'*G`. This computation utilizes randomized Jacobian point
```

Comment:
I'm not sure I agree. After k' has been computed (which happens in Botan inside of ``EC_Point EC_Point_Base_Point_Precompute::mul()`` if I see it correctly, then afterwards the usual scalar multiplication is performed. I don't exactly get what "Jacobian point coordinates" should mean here (I think Jacobians don't have to be mentioned here as Elliptic Curves are their own Jacobians and are Abelian Varieties or what is the reason for a Jacobian being mentioned here?) and the randomization is described in 2 as far as I can see.

Reply:
I suggest to just remove all after ``Compute point :math:`W=(x_1,y_1)=k'*G`.``.

Don't know either what the original author meant here with *Jacobian point coordinates*.

Reply:
Removed the confusing part.